composer.json

```diff
@@ -18,11 +18,12 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": "~2.0.14",
+        "yiisoft/yii2": "~2.0.38",
         "ext-mongodb": ">=1.0.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.8.27|~5.7.21|^6.2"
+        "phpunit/phpunit": "4.8.27|~5.7.21|^6.2",
+        "yiisoft/yii2-dev": "~2.0.38"
```

Comment:
Why do we need this dependency?

Reply:
I might have used it for testing, but can't think of any reason at the moment -> removed.